frame/bridge/bsc/src/benchmarking.rs

```diff
@@ -54,25 +52,4 @@ benchmarks! {
 		let caller: T::AccountId = T::AccountId::decode(&mut &[0; 32][..]).unwrap_or_default();
 
 	}:_(RawOrigin::Signed(caller), header)
-
-	submit_header {
-		let genesis_header = BscHeader::decode(&mut hex2bytes_unchecked(GENSIS_HEADER_BYTES).as_slice()).unwrap();
-		let update_headers = Vec::<BscHeader>::decode(&mut hex2bytes_unchecked(VERIFIED_HEADERS_BYTES).as_slice()).unwrap();
-		let submited_header = BscHeader::decode(&mut hex2bytes_unchecked(SUBMIT_HEADER_BYTES).as_slice()).unwrap();
-		let initial_authority_set =
-				<BscBridge<T>>::extract_authorities(&genesis_header).unwrap();
-
-		Authorities::<T>::put(&initial_authority_set);
-		FinalizedAuthorities::<T>::put(&initial_authority_set);
-		FinalizedCheckpoint::<T>::put(&genesis_header);
-		AuthoritiesOfRound::<T>::insert(
-			&genesis_header.number / T::BscConfiguration::get().epoch_length,
-			(0u32..initial_authority_set.len() as u32).collect::<Vec<u32>>(),
-			);
-		let caller: T::AccountId = T::AccountId::decode(&mut &[0; 32][..]).unwrap_or_default();
-		assert_ok!(<BscBridge<T>>::relay_finalized_epoch_header(
```

Comment:
Should we move this to unit tests?

@xiaoch05 Please help check this, are we good to remove it?

Reply:
Yes, we can remove it now.